api/src/scripts/populate_db.py

```diff
@@ -49,11 +49,19 @@ class DatabasePopulateHelper:
     Helper class to populate the database
     """
 
-    def __init__(self, filepath):
+    def __init__(self, filepaths):
```

Comment:
Add a comment to let the consumer knows that accepts a list and also a single _str_